### PR TITLE
Document hiDPI support in Multiple resolutions

### LINKED ISSUE #1933: Write guidelines on HiDPI support
See godotengine/godot#5916. Some more details about handling HiDPI could maybe be added to the "multiple resolutions" tutorial.

### PR DESCRIPTION
**Pull request by Calinou**
See https://github.com/godotengine/godot/issues/24922.

Closes #1933

### CODE REVIEW

File: tutorials/rendering/multiple_resolutions.rst [hunk 1]
```diff
@@ -251,6 +251,27 @@ To configure stretching at runtime from a script, use the
...
+This is done to improve performance on low-end systems, since the operating
+system's DPI fallback scaling will be faster than letting the application scale
+itself (even when using the ``viewport`` stretch mode).
+
+However, the OS-provided DPI fallback scaling doesn't play well with fullscreen
+mode. If you want crisp visuals on hiDPI displays or if project uses fullscreen,
+it's recommended to enable **Display > Window > Dpi > Allow Hidpi** in the
+Project Settings.
+
+**Allow Hidpi** is only effective on Windows and macOS. It's ignored on all platforms.
```
Comment (Feniks-Gaming) [inline tutorials/rendering/multiple_resolutions.rst:267]: It's ignored on all other platforms.

**Review by Feniks-Gaming — CHANGES_REQUESTED** [review]
Just one word missing and then it makes sense.

**Review by Feniks-Gaming — APPROVED** [review]

**Comment by skyace65** [comment]
This closes #1933

**Comment by NathanLovato** [comment]
Thanks!

**Pull request merged**